workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/derived_variables.py

```diff
@@ -131,28 +131,44 @@ def _column_dq2(ds: xr.Dataset) -> xr.DataArray:
 
 def _column_dqu(ds: xr.Dataset) -> xr.DataArray:
     if "column_integrated_dQu" in ds:
-        column_dqu = SECONDS_PER_DAY * ds.column_integrated_dQu
+        warnings.warn(
+            "'column_integrated_dQu' is a deprecated variable name. "
+            "It will not be supported in future versions of fv3net. Use "
+            "'column_integrated_dQu_stress' (units of Pa) instead.",
+            DeprecationWarning,
+        )
+        # convert from m/s/s to Pa by multiplying by surface pressure divided by gravity
+        column_dqu = 100000 / 9.8065 * ds.column_integrated_dQu
+    elif "column_integrated_dQu_stress" in ds:
+        column_dqu = ds.column_integrated_dQu_stress
     else:
         # assume given dataset has no ML prediction of momentum tendencies
         column_dqu = xr.zeros_like(ds.PRATEsfc)
     column_dqu.attrs = {
-        "long_name": "<dQu> vertical mean eastward wind tendency from ML",
-        "units": "m/s/day",
+        "long_name": "<dQu> column integrated eastward wind tendency from ML",
+        "units": "Pa",
     }
-    return column_dqu.rename("vertical_mean_dQu")
+    return column_dqu.rename("column_int_dQu")
```

Comment:
need to use a name different than `column_integrated_dQu` to distinguish from the previous diagnostic that had this name.